src/main/java/com/powsybl/openloadflow/network/impl/LfVscConverterStationImpl.java

```diff
@@ -78,12 +79,14 @@ public double getTargetQ() {
 
     @Override
     public double getMinP() {
-        return -getStation().getHvdcLine().getMaxP() / PerUnit.SB;
+        HvdcLine hvdcLine = getStation().getHvdcLine();
+        return hvdcLine != null ? -hvdcLine.getMaxP() / PerUnit.SB : -Double.MAX_VALUE;
```

Comment:
It is not needed as you cannot have a VSC without any HVDC line in network loading.

Reply:
I had a failing test without adding it.

Reply:
ok, see #691 for a better solution about hvdc line.